bind/motor-mobile/connect.go

```diff
@@ -0,0 +1,59 @@
+package motor
+
+import (
+	"fmt"
+
+	ct "github.com/sonr-io/sonr/third_party/types/common"
+	mt "github.com/sonr-io/sonr/third_party/types/motor/api/v1"
+	_ "golang.org/x/mobile/bind"
+)
+
+func Connect(buf []byte) ([]byte, error) {
```

Comment:
Is there a reason the bullk of the implementation is in `bind` and not within `pkg/motor` for connection?